benchmark/tests/test_webui.py

```
@@ -274,33 +281,27 @@ def test_prepare_run_config_merges_missing_bundled_skills_with_custom_skills(tmp
-    assert 'trigger_mode = "manual"' in rendered
-    assert "max_iterations = -1" in rendered
-    assert "screenshot_keep_n = 3" in rendered
-    assert config["model_providers"]["benchmark"]["type"] == "openrouter"
-    assert config["model"]["provider"] == "benchmark"
-    assert config["model"]["model"] == "qwen3.6-35b"
-    assert "temperature = 0.2" in rendered
-    assert "max_response_tokens = 1000" in rendered
-    assert "voice_streaming_tts_enabled = false" in rendered
-    assert "voice_tool_call_speech = false" in rendered
-    assert "voice_progress_speech_enabled = false" in rendered
+    with pytest.raises(FileNotFoundError, match="agent.toml"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)